src/org/usfirst/frc/team1492/robot/DriveBase.java

```diff
@@ -5,15 +5,15 @@
 import edu.wpi.first.wpilibj.VictorSP;
 import edu.wpi.first.wpilibj.interfaces.Gyro;
 
-public class DriveBase {
+class DriveBase {
 
-    VictorSP driveLeft;
-    VictorSP driveRight;
-    Solenoid shifter;
+    private VictorSP driveLeft;
+    private VictorSP driveRight;
+    private Solenoid shifter;
     
-    ADXRS450_Gyro gyro;
+    private ADXRS450_Gyro gyro;
```

Comment:
The `VictorSP`s and the `ADXRS450_Gyro` should be private; the rest should be unchanged.